src/main/java/com/checkout/payments/four/request/source/apm/RequestSofortSource.java

```diff
@@ -13,6 +16,19 @@
 @ToString(callSuper = true)
 public final class RequestSofortSource extends AbstractRequestSource {
 
+    @SerializedName("countryCode")
```

Comment:
Are these ones necessary when the attribute is already in camelCase?

Reply:
Good question.
Brief context: we are using gson library to serialize and deserialize objects in Java, here is the [implementation](https://github.com/checkout/checkout-sdk-java/blob/master/src/main/java/com/checkout/GsonSerializer.java)

Basically what happen in background, whenever we send an object, the serializer full map the property for example
name->countryCode -> serialized -> country_code

At one point we started to enforce the serialization if for some reason we change the library or something, so in this case we must send the property as `countryCode` because the API expects the name as it is.

That's the reason of adding this serialized name.


![image](https://user-images.githubusercontent.com/30295119/167215521-11847d70-5f32-479b-9033-c0cd34222658.png)